pkg/controllers/hub/trafficmanagerbackend/controller.go

```diff
@@ -505,7 +505,7 @@ func generateAzureTrafficManagerEndpoint(backend *fleetnetv1alpha1.TrafficManage
 	endpointName := fmt.Sprintf(AzureResourceEndpointNameFormat, generateAzureTrafficManagerEndpointNamePrefixFunc(backend), backend.Spec.Backend.Name, service.Spec.ServiceReference.ClusterID)
 	return armtrafficmanager.Endpoint{
 		Name: &endpointName,
-		Type: ptr.To(string(armtrafficmanager.EndpointTypeAzureEndpoints)),
+		Type: ptr.To(string("Microsoft.Network/trafficManagerProfiles/" + armtrafficmanager.EndpointTypeAzureEndpoints)),
```

Comment:
Is this required by ARM traffic manager?

Reply:
yes, it's required by atm call. Otherwise, call will fail as 400.